Chatdown/lib/index.js

```diff
@@ -35,6 +35,9 @@ let workingDirectory;
  * @returns {Promise<Array>} Resolves with an array of Activity objects.
  */
 module.exports = async function readContents(fileContents, args = {}) {
+    if (args.static || args.s) {
+        now = Number(new Date(2015, 9, 15, 12, 0, 0, 0));
```

Comment:
Is this hard-coded date intentional?
If so, use the `getTime()` function instead of a cast to a number. e.g.
```js
now = new Date(2015, 9, 15, 12, 0, 0, 0).getTime():
```

Reply:
yes it is, it's the date the BF project was started. :)